src/CMakeLists.txt

```diff
@@ -229,6 +229,7 @@ ncnn_add_layer(Reshape)
 ncnn_add_layer(ROIPooling)
 ncnn_add_layer(Scale)
 ncnn_add_layer(Sigmoid)
+ncnn_add_layer(HardSigmoid)
```

Comment:
new layer should be appended at the end of this list, for layer type index compability